src/tyr/route_serializer_osrm.cc

```diff
@@ -283,23 +206,85 @@ json::MapPtr geojson_shape(const std::vector<PointLL> shape) {
   return geojson;
 }
 
+// Maps Valhalla edge and road class to an OpenLR form-of-way.
+FormOfWay RoadClassToFOW(const TripLeg::Node& node) {
```

Comment:
Could we add `FoW` as a member to the protobuf trip edge and set it in triplegbuilder. That way all downstream uses of it wil be supported? There is a function in that source file called `AddTripEdge` i think it could be added there.